spec/definitions/ExportSchedule.yaml

```diff
@@ -0,0 +1,95 @@
+type: object
+required:
+  - format
+  - name
```

Comment:
`format` & `name` are not valid fields below. They have been moved into `dataExportInstruction`